pkg/certificates/reconciler/certificates.go

```diff
@@ -41,8 +41,17 @@ const (
 	expirationInterval   = time.Hour * 24 * 30       // 30 days
 	rotationThreshold    = 10 * time.Minute
 
+	// certificates used by control elements such as autoscaler, ingress controller
 	controlPlaneSecretType = "control-plane"
-	dataPlaneSecretType    = "data-plane"
+
+	// certificates used by trusted data routing elements such as activator, ingress gw
+	dataPlaneRoutingSecretType = "data-plane-routing"
```

Comment:
Could we drop the `routing` part, isn't it redundant with `dataPlane`? Also the ingress-gateway (comment) not sure the ingress-gateway can actually use these certs.

Reply:
We have a data-plane and a control-plane. I think there is a common understanding of those in Knative. Data-plane is responsible for routing. Is there a second certificate for the data-plane? If not `dataPlaneSecretType` should be clear enough, right?

Reply:
I prefer "data-plane-internal" if these certificates are not meant to be touched by the user. "Routing" just adds some info that does not help the users mental model about the setup much. An "edge" component (linking to the other comment) can do routing too.

Reply:
> I prefer "data-plane-internal" or "data-plane-system" if these certificates are not meant to be touched by the user. This is also aligned with what Evan suggested above. "Routing" just adds some info that does not help the users mental model about the setup much. An "edge" component (linking to the other comment) can do routing too.

"data-plane-internal" is very confusing   - is queue not internal? We use to call the entire encryption solution internal. 

"data-plane-system" - makes it impossible to discuss multiple instances of "data-plane-system"s and a unique "system-id" to each one...

Maybe we can simply call it "data-plane-ingress" ? and each instance has an "ingress-id"? Note that I now refer to the activator part of the "data-plane-ingress".

Reply:
@evankanderson commented that "channel" may confuse people due to eventing channels.  

This is the diagram I have in mind:
![image](https://user-images.githubusercontent.com/4232801/229730577-7b66f780-9abd-4a8e-8653-463ab1241039.png)
Given this diagram - 

- Box of type X sends
- Box of type X receives
- Box of type Y routes

Therefore I still think X is best called "edge", but can be called "user" os something along this line.
Y  is best called "Router" but can may also be called "Channel" or something in this area.

Reply:
I'd agree with David that we want to distinguish between the multi-tenant components ("routing") and the components co-located with user workloads ("user").  It's possible we're over-reacting to the previous sharing of SANs, but I'd prefer to have distinct names here.